app/models/manageiq/providers/ibm_cloud/container_manager.rb

```diff
@@ -0,0 +1,201 @@
+class ManageIQ::Providers::IbmCloud::ContainerManager < ManageIQ::Providers::Kubernetes::ContainerManager
+  require_nested :Container
+  require_nested :ContainerGroup
+  require_nested :ContainerNode
+  require_nested :EventCatcher
+  require_nested :EventParser
+  require_nested :Refresher
+  require_nested :RefreshWorker
+
+  def self.ems_type
+    @ems_type ||= "iks".freeze
+  end
+
+  def self.description
+    @description ||= "IBM Cloud Kubernetes Service".freeze
+  end
+
+  def self.display_name(number = 1)
+    n_('Container Provider (IBM Cloud)', 'Container Providers (IBM Cloud)', number)
+  end
+
+  def self.default_port
+    443
```

Comment:
Is 443 the default port for the API server?

Reply:
The port that was used on my cluster was `31821`, but I'm not sure if `443` is the default
i.e.
```
apiVersion: v1
clusters:
- cluster:
    certificate-authority: <ca-path>
    server: https://c103.ca-tor.containers.cloud.ibm.com:31821
  name: <cluster-name>
```

Reply:
Yeah I don't think that IKS has a default port so best to leave this out and not have a default port in the params_for_create method below